.github/pull_request_template.md

```diff
@@ -1,62 +1,83 @@
-### Description
-
-> Describe your PR! If this PR closes an issue, use “Fixes #(issue_number)" syntax so GitHub will auto-close it when merged.
-
-### Select what type of change this PR introduces:
+<!--
+  Hi, thanks for contributing to Wasp!
 
-1. [ ] **Just code/docs improvement** (no functional change).
-2. [ ] **Bug fix** (non-breaking change which fixes an issue).
-3. [ ] **New feature** (non-breaking change which adds functionality).
-4. [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).
+  Comments like this one won't be shown in the final PR, but they contain
+  instructions to guide you.
 
-### Update Waspc ChangeLog and version if needed
+  Make sure to follow this PR template, so that we can speed up the review process.
+  It will also help you not forget important steps when making a change.
 
-If you did a **bug fix, new feature, or breaking change**, that affects `waspc`, make sure you satisfy the following:
+  If you don't know how to fill any of the sections below, it's okay to leave them
+  blank, we will help you out during the review.
+-->
 
-1. [ ] I updated [`ChangeLog.md`](https://github.com/wasp-lang/wasp/blob/main/waspc/ChangeLog.md) with description of the change this PR introduces.
-2. [ ] I bumped `waspc` version in [`waspc.cabal`](https://github.com/wasp-lang/wasp/blob/main/waspc/waspc.cabal) to reflect changes I introduced, with regards to the version of the latest wasp release, if the bump was needed.
-
-### Add a regression test if needed
+### Description
 
-If you did a **bug fix**, make sure you satisfy the following:
+<!--
+  Describe your PR!
 
-1. [ ] I added a regression test that reproduces the bug and verifies the fix.
+  Common questions we'd like you to answer:
+  - What's the motivation for this change?
+  - Which changes are included in this PR?
+    - If there are many different changes, consider splitting your PR into smaller
+      ones. It will go through faster!
 
-If you're unable to add a regression test, please explain why.
-This likely indicates that our current testing setup needs improvement.
+  If this PR closes an issue, write “Fixes #XXXX" so GitHub will link them together.
 
-### Test Coverage
+  You can also answer some of these questions if they are relevant:
+  - Does this change affect users? How?
+  - Have you considered any other approaches? Why is this one the best?
+  - Are there any drawbacks or edge cases?
+  - What are the possibilities for future work?
+-->
 
-Please ensure your changes are adequately tested:
+### Select what type of change this PR introduces:
 
-1. [ ] **My changes are covered by tests** (unit, integration, or e2e tests as appropriate).
+<!-- Put an x in between the brackets to select options, like so: [x] -->
```

Comment:
Oh ok you went really into explaining, do we want to go taht low, or shoudl we assume they know how to checkmark box in MD? Although, I do like that it is very clear what I have to do here hm. But it does introduce some noise. Not sure myself.